UIViewController+Alert.m

```diff
@@ -32,6 +32,20 @@ - (void)showActionSheetWithTitle:(NSString *)title message:(NSString *)message o
     [self showActionSheetWithTitle:title message:message options:options handlers:handlers presentFromView:nil];
 }
 
+- (void)showiPadActionSheetWithTitle:(NSString *)title message:(NSString *)message options:(NSArray *)options handlers:(NSArray *)handlers presentFromTable:(UITableView *)presentingTable withIndexPath:(NSIndexPath *)indexPath {
+    CGRect cellRect = [presentingTable cellForRowAtIndexPath:indexPath].frame;
+    cellRect.origin.y += cellRect.size.height * 2;
+    cellRect.origin.y -= presentingTable.contentOffset.y;
+    cellRect.size.height = 1;
```

Comment:
don't want to hardcode in where we're presenting from as it might differ per person, so use the `presentingRect` parameter in the new method signature to pass this in